web/src/components/Plugin/locales/zh-CN.ts

```diff
@@ -21,6 +21,15 @@ export default {
   'component.step.select.pluginTemplate.select.option': '手动配置',
   'component.plugin.pluginTemplate.tip1': '1. 若路由已配置插件，则插件模板数据将与已配置的插件数据合并。',
   'component.plugin.pluginTemplate.tip2': '2. 插件模板相同的插件会覆盖掉原有的插件。',
+  'component.plugin.transformation': '信息转换器',
```

Comment:
This translation is referred to other products.

Reply:
why we need this? No plugins belong to this category

Reply:
You are right, after adjusting plugin categories, no plugins belong to it any more.